### PR TITLE
Add a FAQ item about why Godot uses Vulkan/OpenGL instead of Direct3D

### PR DESCRIPTION
**Pull request by Calinou**
This is a relatively common question in various community channels.

### CODE REVIEW

File: about/faq.rst [hunk 1]
```diff
@@ -183,6 +183,30 @@ free and open-source integration, consider starting the integration work yoursel
...
+Why does Godot use Vulkan or OpenGL instead of Direct3D?
+--------------------------------------------------------
+
+This question usually stems from OpenGL having a bad reputation in the 2000s for
+poor driver compatibility and lower performance. However, many popular
+OpenGL-based games have proven to be well-optimized in practice.
+
+OpenGL drivers have gotten much better over the recent years. While AMD graphics
+drivers on Windows have historically suffered from lower OpenGL performance,
+this is being remedied by Vulkan which is a lower-level API with fewer
```
Comment (clayjohn) [inline about/faq.rst:195]: "this has been remedied by Vulkan"

File: about/faq.rst [hunk 2]
```diff
@@ -183,6 +183,30 @@ free and open-source integration, consider starting the integration work yoursel
...
+driver-specific quirks.
+
+Direct3D is specific to Windows and Xbox, whereas Vulkan and OpenGL can run
+almost anywhere. (Vulkan is supported on macOS and iOS thanks to MoltenVK.)
+
+Since Godot only has a few people working on its renderer, we would prefer
+having fewer rendereing backends to maintain. On top of that, using a single API
```
Comment (clayjohn) [inline about/faq.rst:202]: "rendering" typo

File: about/faq.rst [hunk 3]
```diff
@@ -183,6 +183,30 @@ free and open-source integration, consider starting the integration work yoursel
...
+--------------------------------------------------------
+
+This question usually stems from OpenGL having a bad reputation in the 2000s for
+poor driver compatibility and lower performance. However, many popular
+OpenGL-based games have proven to be well-optimized in practice.
+
+OpenGL drivers have gotten much better over the recent years. While AMD graphics
+drivers on Windows have historically suffered from lower OpenGL performance,
+this has been remedied by Vulkan which is a lower-level API with fewer
+driver-specific quirks.
```
Comment (akien-mga) [inline about/faq.rst:196]: I don't think this kind of background on how OpenGL used to be bad is useful here. We don't need to try to justify why users might ask this question, we just have to answer it.

IMO what users need to know is that Godot's aims for cross-platform compatibility and open standards first and foremost. OpenGL and Vulkan are the technologies that are both open and available (nearly) on all platforms, and therefore that's why we give them the priority. A game developed with Godot on Windows will run out of the box on Linux and macOS thanks to this design decision.

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**